paasta_tools/instance/kubernetes.py

```diff
@@ -707,15 +751,24 @@ async def get_versions_for_replicasets(
     kube_client: kubernetes_tools.KubeClient,
     service: str,
     instance: str,
-    namespace: str,
+    namespaces: Iterable[str],
     pod_status_by_replicaset_task: "asyncio.Future[Mapping[str, Sequence[asyncio.Future[Dict[str, Any]]]]]",
 ) -> List[KubernetesVersionDict]:
-    replicaset_list = await kubernetes_tools.replicasets_for_service_instance(
-        service=service,
-        instance=instance,
-        kube_client=kube_client,
-        namespace=namespace,
-    )
+
+    replicaset_list: List[V1ReplicaSet] = []
```

Comment:
normally i'd suggest that we should call this `replicasets` to not duplicate the type in both the typehint + the variable name, but this was already existing :p